docs/release-notes/1.9.0.md

```diff
@@ -1,4 +1,6 @@
-- Update the Istio version to 1.23.2 [#1024](https://github.com/kyma-project/istio/pull/1024). Read [Istio 1.23.2 Release Announcement](https://istio.io/latest/news/releases/1.23.x/announcing-1.23.2/).
+- During the Istio upgrade, Pods with the Istio sidecar proxies are now divided into smaller groups and restarted in multiple reconciliations instead of all at once. This increases the stability and reliability of the reconciliation for the Istio module's operator. See the [issue #155](https://github.com/kyma-project/istio/issues/155).
+- We've updated the Istio version to 1.23.2 ([#1024](https://github.com/kyma-project/istio/pull/1024)). Read Istio 1.23.2 [Release Announcement](https://istio.io/latest/news/releases/1.23.x/announcing-1.23.2/) and Istio 1.23.0 [Change Notes](https://istio.io/latest/news/releases/1.23.x/announcing-1.23/change-notes/). Take note of the following breaking changes included in the new minor version of Istio:
+  - **Deferred cluster creation**: In the context of Envoy, a cluster is a group of logically similar upstream hosts that Envoy connects to. Typically, clusters are defined and created at startup. This means that regardless of whether or not a cluster will be used during the lifetime of the Envoy process, it will still be initialized. Envoy has introduced a new optimization that allows these clusters to be created on the worker threads inline during requests, which can save memory and CPU cycles. If you rely on the old behavior, consider using the [compatibility mode](https://kyma-project.io/#/istio/user/00-10-overview-istio-controller?id=compatibility-mode), which disables the deferred cluster creation feature by setting **ENABLE_DEFERRED_CLUSTER_CREATION** to `false`.
+  - **Updates to Envoy cluster metrics parsing**: Previously, the Envoy cluster metrics for services that did not use the `.svc.cluster.local` suffix were incorrectly truncated and parsed. This was becuase Envoy cluster metrics use dots `.` as a delimiter between metric namespaces. It is impossible to properly distinguish between those delimiters and the dots in hostnames or cluster names. To address this, the regex for parsing **cluster_name** has been updated to look for a semicolon that indicats the end of a cluster's name. If you have any dependency to the full stat name for cluster metrics, you must update your monitoring system to account for this change. If you require more time to make adjustments, consider using the [compatibility mode](https://kyma-project.io/#/istio/user/00-10-overview-istio-controller?id=compatibility-mode), which reverts this behavior by setting **ENABLE_DELIMITED_STATS_TAG_REGEX** to `false`.
```

Comment:
```suggestion
  - **Updates to Envoy cluster metrics parsing**: Previously, the Envoy cluster metrics for services that did not use the `.svc.cluster.local` suffix were incorrectly truncated and parsed. This was because Envoy cluster metrics use dots `.` as a delimiter between metric namespaces. It is impossible to properly distinguish between those delimiters and the dots in hostnames or cluster names. To address this, the regex for parsing **cluster_name** has been updated to look for a semicolon that indicates the end of a cluster's name. If you have any dependency on the full stat name for cluster metrics, you must update your monitoring system to account for this change. If you require more time to make adjustments, consider using the [compatibility mode](https://kyma-project.io/#/istio/user/00-10-overview-istio-controller?id=compatibility-mode), which reverts this behavior by setting **ENABLE_DELIMITED_STATS_TAG_REGEX** to `false`.
```